financialaid/models.py

```diff
@@ -10,20 +9,40 @@
 from jsonfield import JSONField
 
 from courses.models import (
-    CourseRun,
     Program,
 )
 
 
-class Tier(models.Model):
+class TimeStampedModel(models.Model):
+    """
+    Base model for create/update timestamps
+    """
+    created_on = models.DateTimeField(auto_now_add=True)  # UTC
+    updated_on = models.DateTimeField(auto_now=True)  # UTC
+
+    def update(self, **kwargs):
+        """
+        Automatically update updated_on timestamp
+        """
+        update_fields = {"updated_on"}
```

Comment:
We've encountered issues in the past where the timestamp is not updated because someone used queryset.update() (which does not call .save() because .update() happens on the database level without loading the object(s) into memory).